Luminary099/GIMBAL_LOCK_AVOIDANCE.agc

```diff
@@ -56,7 +56,7 @@ WCALC		LXC,1	DLOAD*
 			5
 		STORE	TM		# MANEUVER EXECUTION TIME SCALED AS T2
 		SETGO
-			CALCMAN2	# D(OFF) = CONTINUE MANEUVER
+			CALCMAN2	# 0(OFF) = CONTINUE MANEUVER
 			NEWANGL +1	# 1(ON) = START MANEUVER
 
 # THE FOUR SELECTABLE FREE FALL MANEUVER RATES SELECTED BY
```

Comment:
Line below this doesn't have spaces between the numbers.

R0030 https://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0364.jpg

```diff
- # LOADING RATEINDX WITH 0, 2, 4, 6, RESPECTIVELY
+ # LOADING RATEINDX WITH 0,2,4,6, RESPECTIVELY
```